### PR TITLE
Add spec for Validator Tendermint Performance Metrics (#491)

### LINKED ISSUE #491: Validators: Simple performance metrics hooks
> 2. **Performance measurement hooks
> We need to be able to measure per validator if a validator is useful or not.

# @Vegaklaus’ outline
 My first thought would be
- A function that devies on who contributed to a block derived from chain data (if the signatures certifying a block are in the order of arrival of the corresponding messages, that could be added)
- A validator keeping statistical data on the order and time of tendermint-internal messages arriving and for now logging that data
- point-to-point communication times between validators for reference

# Scope
> Performance measurement to collect data for future phases

See #475. Scope set by @barnabee [here](https://github.com/orgs/vegaprotocol/projects/65#card-55923190)

### PR DESCRIPTION
**Pull request by ashleyvega**
This PR adds a spec for Validator Tendermint Performance Metrics

Closes #491.

### CODE REVIEW

**Comment by ashleyvega** [comment]
@Vegaklaus @barnabee Please take a quick look at this draft PR.

(It's my first spec, so be gentle.)

**Comment by davidsiska-vega** [comment]
@barnabee @Vegaklaus could you please leave some comments on this draft pr... cheers!

**Comment by ashleyvega** [comment]
And @gchaincl  - please review too, since you're Mr Tendermint :smile:.

File: specs/0000-validator-tendermint-metrics.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+Feature name: Validator Tendermint Performance Metrics
+Start date: 2021-03-11
+
+# Acceptance Criteria
```
Comment (qustavo) [inline specs/0000-validator-tendermint-metrics.md:4]: Is this planned to be implemented as an API endpoint of the same node, or it should be an external process that gather the information and crunch the numbers?

Comment (ashleyvega) [reply specs/0000-validator-tendermint-metrics.md:4]: I hope that tendermint does/can provide enough info via API that an external process can gather this info. If not, we could add/improve tendermint API endpoints.

**Comment by gordsport** [comment]
Closing as duplicate of #762

**Pull request closed without merging**